.github/workflows/call_sailbot_workspace_CI.yml

```diff
@@ -0,0 +1,25 @@
+name: Use sailbot workspace CI in other repos
```

Comment:
Same comments for this file as the similar file in https://github.com/UBCSailbot/network_systems/pull/18

Reply:
Except keep `on.schedule`

Reply:
Done, copied network_systems comments. Kept on.schedule